pkgs/tools/inputmethods/fcitx/fcitx-configtool.nix

```diff
@@ -19,6 +19,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ makeWrapper fcitx cmake isocodes gtk3
     gnome3.adwaita-icon-theme ];
 
+  # Point exec_prefix to installed location of fcitx-remote (in the fcitx
+  # package).
+  preConfigure = ''
+    sed -ie '/^set(exec_prefix /d' CMakeLists.txt
+    substituteInPlace config.h.in \
+      --subst-var-by exec_prefix ${fcitx}
+  '';
+
```

Comment:
shoulnd't this be fixed upstream instead ? it seems to be looking for fcitx `find_package(Fcitx 4.2.8 REQUIRED)` so it could use the correct path no ? https://gitlab.com/fcitx/fcitx-configtool/blob/master/CMakeLists.txt#L43

Reply:
I've decided to patch CMakeLists.txt to point exec_prefix to the right place. I'll try to submit this patch upstream, but I don't know how to navigate their merge request workflow.

Reply:
@teto please take a look

Reply:
It works like github you an fork from https://gitlab.com/fcitx/fcitx-configtool and generate a PR there. The fix doesn't look nixos specific so best upstream though we can carry it until then.

Reply:
Opened https://gitlab.com/fcitx/fcitx-configtool/merge_requests/17 upstream.

Reply:
thanks I subscribed to the PR. Let's see how it goes